polyfills/HTMLDetailsElement/detect.js

```diff
@@ -0,0 +1 @@
+"HTMLDetailsElement " in self;
```

Comment:
This file is used as a predicate for an if statement, because of that it needs to be an expression, which means it can't have a semicolon at the end

```suggestion
"HTMLDetailsElement " in self
```